packages/cadl-ranch-specs/http/type/property/value-types/main.tsp

```diff
@@ -119,17 +119,17 @@ interface Duration extends ModelOperations<DurationProperty, "P123DT22H14M12.011
 
 // Test a model with an enum property
 @doc("Enum that will be used as a property for model EnumProperty. Non-extensible.")
-enum InnerEnum {
+union InnerEnum {
```

Comment:
The doc says explicitly this is non-extensible, we should add `@fixed` instead

Reply:
There is a `FixedInnerEnum` after this model. So I tend to think the doc was wrong. I would update the doc.